fontc/src/args.rs

```diff
@@ -38,6 +38,10 @@ pub struct Args {
     /// Glyph names must match this regex to be processed
     #[arg(short, long, default_value = None)]
     pub glyph_name_filter: Option<String>,
+
+    /// Set to false to skip compilation of OpenType Layout features
+    #[arg(long, default_value = "true", action = ArgAction::Set)]
+    pub compile_features: bool,
```

Comment:
total bikeshed, but would it make more sense to have this be `--no-compile-features` or `--`--no-layout` or something? basically having it false by default and then the presence of the flag disables? just feels *slightly* simpler.

Reply:
I've followed the pattern we used for --emit-ir=false and --prefer-simple-glyphs=false (also default=true bool options), I don't remember the exact details but we were having problems at the time to accomplish exactly what you suggested and ended up doing tihs.. @rsheeter may remember

Reply:
okay that makes sense, thanks for the rationale :)

Reply:
maybe this https://github.com/googlefonts/fontc/issues/224#issuecomment-1490600494